### PR TITLE
some small stuff

### PR DESCRIPTION
**Pull request by BratishkaErik**

### CODE REVIEW

File: build.zig [hunk 1]
```diff
@@ -31,7 +29,7 @@ pub fn build(b: *std.Build) void {
     // Creates a step for unit testing. This only builds the test executable
     // but does not run it.
     const main_tests = b.addTest(.{
-        .root_source_file = .{ .path = package_path },
+        .root_source_file = main_module.source_file,
```
Comment (cryptodeal) [inline build.zig:32]: Ahhh, good to know this is how package exports should be done; still getting used to the new package manager! :)

File: src/graph/utils.zig
Comment (cryptodeal) [inline src/graph/utils.zig:1]: Good catch here and thanks for adding the test! 💯

**Review by cryptodeal — APPROVED** [review]
Changes look great; appreciate the PR! 💯

**Comment by BratishkaErik** [comment]
Thanks!

File: build.zig (hunk 1 above)
Comment (BratishkaErik) [reply build.zig:32]: To be honest I don't know what „a proper way” is (there is no official guide unfortunately), I just noticed that they both duplicate same `LazyPath`

**Pull request merged**